libs/nx-remotecache-s3/src/lib/setup-s3-task-runner.ts

```diff
@@ -75,19 +73,25 @@ export const setupS3TaskRunner = async (
         /* eslint-disable @typescript-eslint/naming-convention */
         Bucket: bucket,
         Key: `${prefix}${filename}`,
+
         /* eslint-enable @typescript-eslint/naming-convention */
       });
-      return getStream.buffer(result.Body as Stream);
+
+      if (typeof result.Body === 'undefined') {
+        throw new Error('Could not retrieve file');
+      }
+
+      return result.Body.transformToWebStream() as unknown as NodeJS.ReadableStream;
```

Comment:
For my information, could you explain why we need to call `transformToWebStream` on the returned Body?

Reply:
@robinpellegrims I stumbled on this https://stackoverflow.com/a/74699284 when trying to sort out the type mismatches. Now that the nx-remotecache-custom lib uses streams, we had to update this logic to send the correct type of stream. I am pretty sure this should work correctly, but if you would like to test locally to ensure it works as expected, please do!

Reply:
Cool, thanks, it looks like the way to go indeed.

Reply:
While testing, I discovered that this isn't working, maybe because we're still on node v16: `Error: Readable.toWeb() is not supported. Please make sure you are using Node.js >= 17.0.0, or polyfill is available.`

This seems to work though: `return result.Body as NodeJS.ReadableStream;`